backend/src/main/java/com/woowacourse/ternoko/domain/Type.java

```diff
@@ -1,6 +1,23 @@
 package com.woowacourse.ternoko.domain;
 
+import java.util.Arrays;
+
 public enum Type {
     COACH,
-    CREW
+    CREW;
+
+    public static boolean isCoachType(final String value) {
+        return COACH.name().equals(value);
+    }
+
+    public static boolean isCrewType(final String value) {
+        return CREW.name().equals(value);
+    }
```

Comment:
static 메서드로 enum의 필드마다 메서드를 만들기보단, 자신의 인스턴스에게 매개변수를 보내서 맞는지 확인하는 방식이 어떨까요?

```suggestion
    CREW("crew");

    private String value;

    Type(final String value) {
        this.value = value;
    }

    public boolean matchType(final String value) {
        return this.value.equals(value);
    }
```

Reply:
이렇게 작성하는게 추후에 유지보수성이 더 좋아보이네요 👍🏻